src/pages/Login.vue

```diff
@@ -275,7 +270,7 @@ export default {
     },
     phoneLogin() {
       // invoked for logging in with Phone
-      this.startLoading();
+      this.submitOTPIconConfig.enabled = true;

```

Comment:
i think it is better to have a data property: `isSubmitOTPInProgress` and then, set `submitOTPIconConfig.enabled` = `isSubmitOTPInProgress`. Code makes more sense then.